### PR TITLE
Real-Estate

### PR DESCRIPTION
**Pull request by shubhra-g**

### CODE REVIEW

File: Real_Estate/spec.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Real-Estate
```
Comment (Pranavtechie) [inline Real_Estate/spec.yml:1]: Please update the name of the project to `Real Estate website landing page`

File: Real_Estate/spec.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+tags: ['HTML/CSS']
+starter-files: https://github.com/shubhra-g/Real-Estate
+type: frontend
+level: easy
+cover-image: https://github.com/shubhra-g/Real-Estate/blob/main/designs/Real-Estate-Preview%5BWebsite%5D.png
+short-description: A simple and minimal landing page of a Real Estate Website.
+long-description: |
```
Comment (Pranavtechie) [inline Real_Estate/spec.yml:8]: The long description should contain the details regarding the landing page that the user is gong to developer, Please re-write the entire long description

File: Real_Estate/step0/instructions.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+**In this step, our goal is to create the navbar and with all the basic details**
+
+You have create variables for the basic colors to be used in entire designing , import fonts and go about designing the entire navbar
+
+- Create a class ```navbar-container``` which includes  ```logo``` of "R-state" , a ```menu-icon``` , a unordered list class displaying  home , about , sales and properties.In the end create a ```log-in``` button
```
Comment (Pranavtechie) [inline Real_Estate/step0/instructions.md:5]: Please be thorough when writing instructions and always provide context to what you are referring to

Comment (Pranavtechie) [reply Real_Estate/step0/instructions.md:5]: For example you can point out the logo's path etc. 

You should make use of single backtick when writing an inline codeblock. 

When writing a codeblock you have to make use of triple backticks. Please update this as well

File: Real_Estate/step0/challenges.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+name: "creating elements and navbar"
+stepId: step0
+stepBreakdown:
+    -text: "add the logo"
```
Comment (Pranavtechie) [inline Real_Estate/step0/challenges.yml:4]: Rephrase the challenges to give a clear cut understand, the challenges should be concise in nature, but you can make it up to 5 - 15 words, not 3-5 words.

Comment (Pranavtechie) [inline Real_Estate/step0/challenges.yml:4]: Rephrase the challenges to give a clear cut understand, the challenges should be concise in nature, but you can make it up to 5 - 15 words, not 3-5 words.

File: Real_Estate/step1/challenges.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+name: "creating the section for home and adding a image and sign-up button"
```
Comment (Pranavtechie) [inline Real_Estate/step1/challenges.yml:1]: Please keep the step names as short as possible, they'll be displayed in the sidebar

**Review by Pranavtechie — CHANGES_REQUESTED** [review]
Upload the icon directly to the boilerplate repo instead of an min.css file 

font.txt contains css reset code, which should have been clearly part of the steps mentioned 

Please download the .ttf format files of the font instead of google fonts 

I see that all the images are not in the same format, it is best to keep all the images in webp, jpg or png. Convert them to a single format as per your wish. (This is to avoid any kind of confusion while importing files) 

UPload a screenshot of the mobile design of the page as well, making responsiveness part of the project 


**Please go through all the mentioned changes in the reivew, if you feel that you have a doubt, feel free to ping me directly in this discussion. **

**Comment by shubhra-g** [comment]
how to upload the icon to the boilerplate repo

**Comment by shubhra-g** [comment]
???

**Pull request closed without merging**